docs/installation.md

```diff
@@ -34,12 +33,11 @@
 - Linux: `apt-get install python-protobuf`
```

Comment:
There is no need to install protobuf.
https://github.com/PokemonGoF/PokemonGo-Bot/pull/3670